CHANGELOG.md

```diff
@@ -9,7 +9,14 @@
 
 ### Breaking changes
 
-* Lorem ipsum.
+* Added support for compound sort and distinct queries.
+    - Multiple consecutive calls to sort or distinct compound on each other
+      in the order applied rather than replacing the previous one.
+    - The order that sort and distinct are applied can change the query result.
+    - It is now an error to define an ascending order on a distinct descriptor, if the
+      order is important then a sort should be applied before the distinct.
+    - Applying an empty sort or distinct descriptor is now a no-op, this
```

Comment:
@kneth Yes, one of the objectives of this change is to remember the applied sort/distinct calls, but creating a new query would "forget" previously applied sort/distinct operations. I didn't think this would be a use case, but if needed we could add support for removing the N'th applied sort/distinct or clear all sorts/distincts.